cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -26,12 +28,12 @@ def pytest_generate_tests(metafunc):
     testgen.parametrize(metafunc, argnames, argvalues, ids=idlist, scope="class")
 
 
-@pytest.fixture(scope='class')
+@pytest.fixture(scope='function')
```

Comment:
can you try to shortly explain the need for the scope change for someone unfamiliar with the details ?